manifests/t/Tencent/WeChat/3.3.5.42/Tencent.WeChat.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tencent.WeChat
+PackageVersion: 3.3.5.42
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe
+  InstallerSha256: 96D92B422535E4EDC88E0A6B6467FA783A5192068FC80ACF4845D4F988F8F09E
+  InstallerSwitches:
+    Custom: /NORESTART
```

Comment:
the silent install switch is `/silent` . Tencent changed the default switch of Nullsoft installer, which cause the problem.

Reply:
I'll give it a try, thanks. :)